domain/infosync/info.go

```diff
@@ -264,6 +268,70 @@ func GetTiFlashTableSyncProgress(ctx context.Context) (map[int64]float64, error)
 	return progressMap, nil
 }
 
+func doRequest(ctx context.Context, addrs []string, route, method string, body io.Reader) (bool, error) {
+	var err error
+	var req *http.Request
+	for _, addr := range addrs {
```

Comment:
Why send it to all the addresses?

Reply:
That is what `pd-ctl` is trying to do: send request until there is an HTTP response, in case that some PD servers are down.